iperf.py

```diff
@@ -68,27 +33,6 @@ def __init__(self, tc: TestConfig, ts: TestSettings):
         self.cluster_ip_addr = self.create_cluster_ip_service()
         self.nodeport_ip_addr = self.create_node_port_service(self.port + 25000)
```

Comment:
We need to set these in the perf class as well if there is code in the perf class that expects this fields to exist

Reply:
Yep I agree. Made the change.